src/index.js

```diff
@@ -1,15 +1,24 @@
-
-
 import './index.css';
-
+import React from 'react';
+import ReactDOM from 'react-dom';
+import {BrowserRouter, Route} from "react-router-dom"
+import './index.css';
+import App from './App';
 import * as serviceWorker from './serviceWorker';
-import {rerenderEntireTree} from './render'
-import {state} from './components/redux/State'
+import {state, addPost, updateNewpostText, subscribe} from './components/redux/State'
+
+const rerenderEntireTree = (state) => {
```

Comment:
Consider fixing the indentation like this:
![image](https://user-images.githubusercontent.com/25229823/77856299-d35de780-71fe-11ea-8295-01cf89451a04.png)

Reply:
done